pintracker/stateless/mock.go

```diff
@@ -0,0 +1,74 @@
+package stateless
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ipfs/ipfs-cluster/api"
+	"github.com/ipfs/ipfs-cluster/state"
+	"github.com/ipfs/ipfs-cluster/test"
+
+	cid "github.com/ipfs/go-cid"
+)
+
+var pinOpts = api.PinOptions{
+	ReplicationFactorMax: -1,
+	ReplicationFactorMin: -1,
+}
+
+type mockState struct {
+	slow bool
+}
+
+// NewMockState return a mock implementation of State.
+func NewMockState(slow bool) state.ReadOnly {
+	return mockState{slow: slow}
+}
+
+func (st mockState) List(ctx context.Context) ([]*api.Pin, error) {
```

Comment:
> slow does not make things slow.

`slow` only make pin and unpin slow, which is done by rpc client. But normal rpc client and slow rpc client returns different results on `Pins` and `PinGet`. `slow` var here is to tell which result should be sent.

Reply:
I guessed that this would be comparatively big. I am doing this separately in https://github.com/ipfs/ipfs-cluster/pull/958, to keep this PR small and keep it just about removing mappintracker.